doc/whats_new/v0.24.rst

```diff
@@ -44,6 +44,15 @@ Changelog
     :pr:`123456` by :user:`Joe Bloggs <joeongithub>`.
     where 123456 is the *pull request* number, not the issue number.
 
+:mod:`sklearn.decomposition`
+............................
+
+- |Enhancement| added a new approximate solver (randomized SVD, available with
```

Comment:
please also indicate the change in 'auto'

Reply:
agreed